IM.py

```diff
@@ -0,0 +1,56 @@
+"""The purpose of this file is to illustrate how to process Spotter wave spectra to calculate wind speed using the
+IM (inverse modelling) method in Dorsay et al. 2023."""
+
+from roguewave import load
+from roguewave.wavephysics.windestimate import estimate_u10_from_source_terms
+from roguewave.wavephysics.balance import create_balance
+import matplotlib.pyplot as plt
+import pandas as pd
+
+spotter_id = "SPOT-010340"
+
+### Define method parameters
+im_params = load('s3://sofar-wx-data-dev-os1/proxy_wind/calibration/final_calibration_st4_weighted=True.zip')
```

Comment:
We should add these files to the repo - we don't want to give outsiders axis to the bucket. I would also rename the files to 'im_calibration.zip', 'S2022_calibration.zip' etc